rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -2469,6 +2450,29 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
         )
     }
 
+    def authProjectsToCreateAjax() {
+        AuthContext authContext = rundeckAuthContextProcessor.getAuthContextForSubjectAndProject(
```

Comment:
I think there is no need for "getAuthContextForSubjectAndProject", probably just "getAuthContextForSubject", since this endpoint does not require a project parameter

Reply:
changed it to only retrieve auth context by subject